css-canvas-medical/styles.css

```diff
@@ -0,0 +1,254 @@
+  width: 110px;
+  color: gray;
+  font-size: 12px;
+}
+
+.container {
+  max-width: 800%;
+}
+
+.column-full {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.column-half {
+  flex-wrap: wrap;
+  width: 50%;
+}
+
+.logo {
+  margin-right: 250px;
+  width: 120px;
+}
+
+.image {
+  background-size: cover;
+  background-image: url("images/hero.jpg");
+  height: 390px;
+  width: 700px;
+  margin-top: 35px;
+  position: relative;
+}
+
+.image-text {
+  position: absolute;
+  font-family: "Tiempos Headline", sans-serif;
+  font-size: 25px;
+  color: white;
+  width: 350px;
+  top: 50px;
+  left: 30px;
+}
+
+.p-image-text {
+  position: absolute;
+  font-family: avenir, sans-serif;
+  font-size: 13px;
+  color: white;
+  width: 360px;
+  top: 140px;
+  left: 30px;
+}
+
+.overlay {
+  background: radial-gradient(transparent, rgb(0 0 0 / 25%));
+  position: absolute;
+  bottom: 0;
+  top: 0;
+  left: 0;
+  right: 0;
+}
+
+.contact {
+  border: none;
+  border-radius: 5px;
+  color: white;
+  background-color: rgb(18 58 111);
+  width: 90px;
+  height: 40px;
+}
+
+.button-image {
+  position: absolute;
+  padding: 10px;
+  width: 135px;
+  border: none;
+  border-radius: 5px;
+  color: white;
+  background-color: rgb(18 58 111);
+  bottom: 35px;
+  left: 30px;
+}
+
+.article-body {
+  background-color: rgb(244 245 245);
+  position: relative;
+  width: 60rem;
+  height: 23rem;
+  padding: 20px;
+  margin-top: 40px;
+}
+
+.column-half-body {
+  text-align: center;
+  color: rgb(71 71 71);
+  margin-left: auto;
+  margin-right: auto;
+  width: 350px;
+}
+
+.column-one-third-body {
+  position: absolute;
+  color: gray;
+  left: 160px;
+  bottom: 70px;
+}
+
+.column-one-third-bodytwo {
+  position: absolute;
+  color: gray;
+  right: 150px;
+  bottom: 75px;
+}
+
+.p-mainbody-text {
+  color: gray;
+  font-size: 13px;
+  margin-bottom: 2px;
+  margin-top: -25px;
+}
+
+.column-one-fourth {
+  font-family: "Tiempo Headline", sans-serif;
+  width: 35%;
+}
+
+.article-bottom {
+  display: flex;
+  justify-content: center;
+  position: relative;
+  width: 60rem;
+  height: 8rem;
+}
+
+.bottom-text {
+  flex-wrap: wrap;
+  position: absolute;
+  color: rgb(71 71 71);
+  font-size: 25px;
+  width: 300px;
+  left: 75px;
+}
+
+.p-bottom-text {
+  position: absolute;
+  width: 300px;
+  font-size: 13px;
+  color: gray;
+  top: 60px;
+  left: 75px;
+}
+
+.bottom-button {
+  padding: 10px;
+  width: 135px;
+  border: none;
+  border-radius: 5px;
+  color: white;
+  background-color: rgb(18 58 111);
+  height: 40px;
+  margin-top: 40px;
+  margin-left: 300px;
+}
+
+.article-info {
+  font-family: "Tiempo Headline", sans-serif;
+  display: flex;
+  position: relative;
+  background-color: gray;
+  width: 60rem;
+  height: 20rem;
+}
+
+.column-one-fifth {
+  flex-wrap: wrap;
+  width: calc(100%/ 5);
+  margin: 100px 70px;
+  color: white;
+}
+
+.column-one-fifth-info {
+  position: absolute;
```

Comment:
you want to avoid using position absolute on columns. columns should be wrapped in rows with display flex on them. Once you use flexbox to get the items close to there position, a little margin and padding will get you what you need :)